python/feninit.py

```diff
@@ -1225,6 +1227,10 @@ def _choosePid(self):
         print 'Enter PID'
         return readinput.call(': ', '-d')
 
+    def _choosePackage(self):
+        print 'Enter package (e.g., org.mozilla.geckoview_example): '
+        return readinput.call(': ', '-d')
```

Comment:
Put the prompt on the same line and don't use the `-d` switch, e.g.

 `return readinput.call('Enter package (e.g., org.mozilla.geckoview_example): ')`